src/main/java/CommandHandler.java

```diff
@@ -0,0 +1,213 @@
+package main.java;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class CommandHandler {
+    Scanner sc;
+    ArrayList<Task> taskList;
+
+    public CommandHandler() {
+        this.sc = new Scanner(System.in);
+        this.taskList = new ArrayList<>();
+    }
+
+    public void handleCommand() {
+        while (true) {
+            String command = sc.next();
+            switch (command) {
+                case "bye":
```

Comment:
There's no indention for the case statements. It should be 'aligned' with the switch statement.